pyanaconda/ui/gui/spokes/root_password.py

```diff
@@ -102,8 +102,6 @@ def initialize(self):
         # check that the content of the password field & the conformation field are the same
         self._confirm_check = input_checking.PasswordConfirmationCheck()
         # regards both fields empty as success to let the user escape
-        self._confirm_check.success_if_confirmation_empty = True
-        # check password validity, quality and strength
```

Comment:
I think you wanted to remove the previous comment

Reply:
Should, be fixed now, thanks! :)